packages/SwingSet/src/kernel/kernel.js

```diff
@@ -710,6 +724,7 @@ export default function buildKernel(kernelEndowments) {
     }
 
     kernelKeeper.setInitialized();
+    commitCrank(); // commit "crank 0"
```

Comment:
This is an (important, but) unrelated bugfix.. let's land this all together now, but next time maybe let's put this into a separate PR